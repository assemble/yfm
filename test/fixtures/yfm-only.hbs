---
title: YFM Only
description: There is no content on this page.
---