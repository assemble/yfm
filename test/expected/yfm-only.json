{
  "metadata": {
    "title": "YFM Only",
    "description": "There is no content on this page."
  },
  "content": "",
  "original": "---\ntitle: YFM Only\ndescription: There is no content on this page.\n---"
}